client/src/resources/lang/en_us.json

```diff
@@ -776,7 +785,16 @@
     "chatMessage.phaseChange":"\\0 \\1",
     "chatMessage.phaseChange.nomination":"\\0 \\1\n\\2 trials left",
     "chatMessage.phaseChange.trial":"\\0 \\1\n\\2 is on trial",
-    "chatMessage.gameOver": "Game Over",
+    "chatMessage.phaseChange.recess":"Recess",
+    "chatMessage.gameOver.conclusion.town": "The town has expelled the evil!",
+    "chatMessage.gameOver.conclusion.mafia": "The syndicate has taken over!",
+    "chatMessage.gameOver.conclusion.cult": "The cult has indoctrinated everyone!",
+    "chatMessage.gameOver.conclusion.fiends": "The fiends have won!",
+    "chatMessage.gameOver.conclusion.politician": "The politician has taken control!",
+    "chatMessage.gameOver.conclusion.draw": "The game has ended in a draw!",
+    "chatMessage.gameOver.player.won.true": "\\0 won! (\\1)",
+    "chatMessage.gameOver.player.won.false": "\\0 lost! (\\1)",
+    "chatMessage.gameOver.player.crumb": "\\0 \\1",

```

Comment:
This is a special kind of torture for me specifically

If you want this PR merged your gonna have to do it without my approval specifically because of this
These messages are so similar to eachother that it barely matters
"X won" would be good enough

Reply:
The reason I made them the same was because different languages might not be able to just slot in the word for win/lose there. I can make the `(\\1)` happen in code though

Reply:
"chatMessage.gameOver.conclusion": "The following word is the word that represents the game conclusion that was reached: \\0"

Im pretty sure all spoken languages can represnt enums right

Reply:
Oh you're talking about those? Oh then I just disagree. I think it's nice to have a custom message for each one. I can add a default message, I guess.

But it's way cooler to see "The syndicate has taken over!" than "The game ended with conclusion Syndicate."